File: src/Models/Inventory.java
package Models;

public class Inventory implements HasID{
    public int inventory_id;
    public int wine_id;
    public int location_id;
    public int quantity;
    public int bottle_size_ml;

    public Inventory(int inventory_id, int wine_id, int location_id, int quantity, int bottle_size_ml) {
        this.inventory_id = inventory_id;
        this.wine_id = wine_id;
        this.location_id = location_id;
        this.quantity = quantity;
        this.bottle_size_ml = bottle_size_ml;
    }

    public int getID() {
        return inventory_id;
    }

    public void setInventory_id(int inventory_id) {
        this.inventory_id = inventory_id;
    }

    public int getWine_id() {
        return wine_id;
    }

    public void setWine_id(int wine_id) {
        this.wine_id = wine_id;
    }

    public int getLocation_id() {
        return location_id;
    }

    public void setLocation_id(int location_id) {
        this.location_id = location_id;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }

    public int getBottle_size_ml() {
        return bottle_size_ml;
    }

    public void setBottle_size_ml(int bottle_size_ml) {
        this.bottle_size_ml = bottle_size_ml;
    }

    @Override
    public String toString() {
        return "Inventory{" +
                "inventory_id=" + inventory_id +
                ", wine_id=" + wine_id +
                ", location_id=" + location_id +
                ", quantity=" + quantity +
                ", bottle_size_ml=" + bottle_size_ml +
                '}';
    }
}

File: Python code/Entities/Inventory.py
class Inventory:
    def __init__(self, inventory_id, wine_id, location_id, quantity, bottle_size_ml):
        self.inventory_id = inventory_id
        self.wine_id = wine_id
        self.location_id = location_id
        self.quantity = quantity
        self.bottle_size_ml = bottle_size_ml

    def get_inventory_id(self):
        return self.inventory_id

    def get_wine_id(self):
        return self.wine_id

    def get_location_id(self):
        return self.location_id

    def get_quantity(self):
        return self.quantity

    def get_bottle_size_ml(self):
        return self.bottle_size_ml

    def set_inventory_id(self, inventory_id):
        self.inventory_id = inventory_id

    def set_wine_id(self, wine_id):
        self.wine_id = wine_id

    def set_location_id(self, location_id):
        self.location_id = location_id

    def set_quantity(self, quantity):
        self.quantity = quantity

    def set_bottle_size_ml(self, bottle_size_ml):
        self.bottle_size_ml = bottle_size_ml

    def __str__(self):
        return f"Inventory: {self.wine_id} ({self.quantity} bottles)"